gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/reactive/handlers/api/adapter/invoker/InvokerAdapter.java

```diff
@@ -46,6 +46,9 @@ public class InvokerAdapter implements HttpInvoker, Invoker, io.gravitee.gateway
 
     private static final Logger log = LoggerFactory.getLogger(InvokerAdapter.class);
     static final String GATEWAY_CLIENT_CONNECTION_ERROR = "GATEWAY_CLIENT_CONNECTION_ERROR";
+    static final String ERROR_KEY_CLIENT_ABORTED_DURING_RESPONSE = "ERROR_CLIENT_ABORTED_DURING_RESPONSE";
```

Comment:
As we have `GATEWAY_CLIENT_CONNECTION` **`_ERROR`** should we rename it 
`CLIENT_ABORTED_DURING_RESPONSE_ERROR` ?